x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality/header.tsx

```diff
@@ -5,12 +5,46 @@
  * 2.0.
  */
 
-import { EuiPageHeader } from '@elastic/eui';
+import {
+  EuiBetaBadge,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiPageHeader,
+  EuiSpacer,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
 import React from 'react';
 import { datasetQualityAppTitle } from '../../../common/translations';
 
+export const betaBadgeLabel = i18n.translate('xpack.datasetQuality.betaBadgeLabel', {
+  defaultMessage: 'Beta',
+});
+
+export const betaBadgeDescription = i18n.translate('xpack.datasetQuality.betaBadgeDescription', {
+  defaultMessage:
+    'This feature is currently in beta. If you encounter any bugs or have feedback, we’d love to hear from you. Please open a support issue and/or visit our discussion forum.',
+});
+
 // Allow for lazy loading
 // eslint-disable-next-line import/no-default-export
 export default function Header() {
-  return <EuiPageHeader bottomBorder pageTitle={datasetQualityAppTitle} />;
+  return (
+    <>
+      <EuiFlexGroup gutterSize="s" responsive={false} alignItems="baseline">
```

Comment:
The section ["Customizing the page header"](https://eui.elastic.co/#/layout/page-header#customizing-the-page-header) in the EUI docs proposes to compose it the other way around. In their example the more elaborate content (i.e. `h2` and the badge) are passed as children to `EuiPageHeader`. I wonder if that could be an approach to better preserve the page structure?

Reply:
Thanks for sharing this, I'll change it to that structure

Reply:
Addressed in [a5c25b7](https://github.com/elastic/kibana/pull/177613/commits/a5c25b78fdab60dd744eff22025be39738be3876).